lib/modules/asset/types/AssetApi.ts

```diff
@@ -142,6 +143,39 @@ export type ApiAssetGetMeasuresResult = {
   total: number;
 };
 
+export interface ApiAssetMeasureIngestRequest extends AssetsControllerRequest {
+  action: "measureIngest";
+
+  assetId: string;
+
+  engineId: string;
+  engineGroup?: string;
+  slotName: string;
+
+  body: {
+    dataSourceId: string;
```

Comment:
Why isn't that a `ApiMeasureSource` instead of just the `dataSourceId` like in `ApiAssetmMeasureIngestRequest`?
```suggestion
    dataSource: ApiMeasureSource;
```

Reply:
@inkedsquid what was the reasons of this difference between single ingest and multiple ingest ?

Reply:
It will always be an `API` datasource, it is inferred by the controller of the ingest endpoints